BuildResidentialHPXML/measure.rb

```diff
@@ -1219,6 +1219,60 @@ def arguments(model)
     arg.setDescription('The conditioned floor area served by the air distribution system.')
     args << arg
 
+    heating_system_type_2_choices = OpenStudio::StringVector.new
+    heating_system_type_2_choices << 'none'
+    heating_system_type_2_choices << HPXML::HVACTypeWallFurnace
+    heating_system_type_2_choices << HPXML::HVACTypeFloorFurnace
+    heating_system_type_2_choices << HPXML::HVACTypeElectricResistance
+    heating_system_type_2_choices << HPXML::HVACTypeStove
+    heating_system_type_2_choices << HPXML::HVACTypePortableHeater
+    heating_system_type_2_choices << HPXML::HVACTypeFireplace
+
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('heating_system_type_2', heating_system_type_2_choices, true)
+    arg.setDisplayName('Heating System 2: Type')
+    arg.setDescription('The type of the second heating system.')
+    arg.setDefaultValue('none')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('heating_system_fuel_2', heating_system_fuel_choices, true)
+    arg.setDisplayName('Heating System 2: Fuel Type')
+    arg.setDescription('The fuel type of the second heating system. Ignored for ElectricResistance.')
+    arg.setDefaultValue(HPXML::FuelTypeElectricity)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('heating_system_heating_efficiency_afue_2', true)
+    arg.setDisplayName('Heating System 2: Rated AFUE')
+    arg.setUnits('AFUE')
+    arg.setDescription('The rated efficiency value of the second Furnace/WallFurnace heating system.')
+    arg.setDefaultValue(0.78)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('heating_system_heating_efficiency_percent_2', true)
+    arg.setDisplayName('Heating System 2: Rated Percent')
+    arg.setUnits('Percent')
+    arg.setDescription('The rated efficiency value of the second ElectricResistance/Stove/PortableHeater/Fireplace heating system.')
+    arg.setDefaultValue(1.0)
+    args << arg
```

Comment:
Since AFUE/Percent have the same units, I suggest combining these into a single `heating_system_heating_efficiency_2` input. No point introducing extra complexity. Then in the code further down, automatically use AFUE or Percent as appropriate based on the heating system type. You can make the same change for the primary heating system.

Reply:
I thought there was a reason we had them separated out. So that different heating system types would get appropriate default values? Why couldn't we just have a single efficiency argument for cooling (even though they're different units), and then set them according to the cooling system type?

Reply:
For cooling system units or heat pump units (SEER/EER, COP/HSPF), the logic definitely makes sense to have different values for different system types because if the default SEER is 13, that wouldn't make sense as a EER value. They are defined differently. In fact, this need to be fixed because you have cooling_system_cooling_efficiency_eer set as 13.

But for heating system units (AFUE/Percent), they are essentially the same. Most users would probably be more confused by treating them differently than vice versa. Also note that the current defaults don't make sense. You have 0.78 for furnaces and 1.0 for other systems. But those default values should be based on _fuel type_, not _system type_. Electric systems will be ~1.0 while fuel systems will generally be in the 0.78 - 0.95 range. Since you have the default fuel type as electric, 0.78 doesn't make sense for an electric furnace. So if you default to electricity, a default of 1.0 can be used for all heating system types; if you default to gas, a default of 0.78 can be used for all heating system types. This needs to be fixed for both primary and secondary heating systems.